src/com/phonebookservice/database/ContactsFileParser.java

```diff
@@ -0,0 +1,82 @@
+package com.phonebookservice.database;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+
+import com.phonebookservice.exception.InternalServerException;
+import com.phonebookservice.model.Contact;
+import com.phonebookservice.server.ContactConverter;
+import com.phonebookservice.util.CollectionUtility;
+
+public class ContactsFileParser implements IContactsParser {
+    private static final String ENCODING = "UTF-8";
+    private final String fileName;
+
+    /**
+     * Initialization of file parser.
+     *
+     * @param fileName the file name.
+     */
+    public ContactsFileParser(final String fileName) {
+        this.fileName = fileName;
+    }
+
+    @Override
+    public final Collection<Contact> readContacts() {
+        final List<Contact> contacts = new ArrayList<>();
```

Comment:
Don't initialize variable on fly.
Initialize it in constructor

Reply:
why ? it is in a method, not for the whole class

Reply:
U t right here, my bad